ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -107,25 +107,6 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
-- as: gcp-ipi-disc-priv-oidc-fips-f60
-  cron: 17 14 25 2,4,6,8,10,12 *
-  steps:
-    cluster_profile: gcp-qe
-    env:
-      ENABLE_OTA_TEST: OCP-30832
```

Comment:
cc @dis016 the case might needed to be changed to another profile accordingly

Reply:
`OCP-30832` is also enabled on other plaforms.

Reply:
ah, great, so we just don't need the case's coverage on gcp, right?

Reply:
Right, the case is independent on platforms.